modules/images-configuration-registry-mirror.adoc

```diff
@@ -191,12 +191,12 @@ unqualified-search-registries = ["registry.access.redhat.com", "docker.io"]
     insecure = false
 ----
 
-.. Pull an image to the node from the source and check if it is actually
-resolved by the mirror.
+.. Pull an image digest to the node from the source and check if it is actually
+resolved by the mirror. The `ImageContentSourcePolicy` supports image digests only, not image tags.
 +
 [source,terminal]
 ----
-sh-4.2# podman pull --log-level=debug registry.access.redhat.com/ubi8/ubi-minimal
+sh-4.2# podman pull --log-level=debug registry.access.redhat.com/ubi8/ubi-minimal@sha256:5cfbaf45ca96806917830c183e9f37df2e913b187adb32e89fd83fa455ebaa6
```

Comment:
In the step 1 , the image used `registry.access.redhat.com/ubi8/ubi-minimal@sha256:c505667389712dc337986e29ffcb65116879ef27629dc3ce6e1b17727c06e78f` , could you update above to make consistent?
 `skopeo copy \
docker://registry.access.redhat.com/ubi8/ubi-minimal@sha256:c505667389712dc337986e29ffcb65116879ef27629dc3ce6e1b17727c06e78f \
docker://example.io/example/ubi-minimal`

Reply:
Thia change has been made and pushed to this branch. Please verify that it is correct.